### ISSUE TITLE
Keyboard focus should be in terminal when launched

### ISSUE THREAD

**Issue by jimmyjones2** [kubernetes-sigs/headlamp#4754]
## Is your feature request related to a problem? Please describe the impact that the lack of the feature requested is creating.
When I launch a terminal in a pod, I need to click on the terminal window before I can type commands. If I type commands before clicking in the terminal, nothing happens. Every time a user launches a terminal, they intend on typing commands into it.

## Describe the solution you'd like
As soon as the terminal is launched, the keyboard focus moves to it.

## What users will benefit from this feature?
This will save a click every time someone launches a terminal. It'll also be a less confusing experience for new users who will need to figure out they need to click into the terminal before they can type commands.

## Are you able to implement this feature?
No

**Comment by illume** [comment]
Seems like this was pretty easy to do... xterm library had a focus() method already.

- https://github.com/kubernetes-sigs/headlamp/pull/4759

Thanks for reporting the good issues!
